### ISSUE TITLE
tabs (10-DAY, etc) no longer available

### ISSUE THREAD

**Issue by rpupkin** [alexis-jaksone/forecast-plus#63]
![wu_extension_02](https://github.com/alexis-jaksone/forecast-plus/assets/33489540/0e070a05-0ea1-4acb-849d-1219d7c5f9a7)
![wu_extension_01](https://github.com/alexis-jaksone/forecast-plus/assets/33489540/6b7691f1-75f6-41f5-9212-239f5bfb356c)

thanks!

**Comment by ziffusion** [comment]
PLEASE bring this back! What is the reason for removing this?

**Comment by CyberAlexP** [comment]
The same.
There are no Today Hourly 10-Day Calendar and other tabs. 
The selected location cannot be added to Favorites.
And the problem is logging into your account from the pop-up window.
I think the problem with what is described here is precisely because of the inability to log into your account.
However, if I open the Weather Underground website, I can easily log into my account. And all the bookmarks are there.
The developers really screwed this up here.

![Weather](https://github.com/alexis-jaksone/forecast-plus/assets/145215289/a5ae8307-4f50-4f59-a9e1-3c68540938c4)

Something like this...

**Comment by samsonite31** [comment]
Yes, please bring this back. It was extremely useful! I've had to open my browser every time I want to see the weather recently. But I love this extension, it's my favorite!
